internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -559,6 +559,26 @@ func StartContainerd(seq runtime.Sequence, data interface{}) (runtime.TaskExecut
 	}, "startContainerd"
 }
 
+func WriteUdevRules(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFunc, string) {
+	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) (err error) {
+		var result *multierror.Error
+
+		rules := r.Config().Machine().Udev().Rules()
+
+		content := ""
+
+		for _, rule := range rules {
+			content += strings.Replace(rule, "\n", "\\\n", 0) + "\n"
+		}
```

Comment:
still need to learn how to do tests, but the thought here is in each rule we will replace each new line with a backslash and a newline. A udev rules file can contain multiple rules. Each rule should be on its own line. If you do want a rule to span multiple lines, you need a backslash before the newline. So this replace on here is to support multi-line rules such as:

```yaml
udev:
  rules:
    - |-
      SUBSYSTEM=="drm",
      KERNEL=="card*",
      GROUP="44",
      MODE="0660"
```